Automate application delivery with Azion Edge Orchestrator (#1280)
aziontech/docs: src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx

---
title: Automate application delivery with Azion Edge Orchestrator
description: >-

>   Learn how to streamline application delivery and infrastructure management using Azion Edge Orchestrator. Enhance performance with efficient edge node management and automated operations.

Reviewer: ```suggestion
  Streamline application delivery and infrastructure management with Azion Edge Orchestrator, enhancing performance with efficient management and automated operations.
```

Reviewer: to be around 160 chars

permalink: /documentation/architectures/deploy/application-delivery/
meta_tags: >-
  Azion Edge Platform, Azion Edge Orchestrator, application delivery automation, edge node management, infrastructure automation, Azion CLI, performance optimization, edge computing
namespace: docs_arch_infra_auto_apps_delivery
---

[Azion Edge Orchestrator](/en/documentation/products/deploy/edge-orchestrator/) enables frequent and reliable application releases through orchestration. This documentation covers the architecture and steps for automated application delivery using Azion Edge Orchestrator, ensuring optimal performance and streamlined operations.

---

> ## Architecture

Reviewer: ```suggestion
## [use case name] Architecture Diagram
```

Reviewer: to align with template

:::note
This flow assumes a service has been created for saving an HTML page containing a list of products. For further information, check the [Implementation](#implementation) section.
:::

![Orchestrator architecture diagram](/assets/docs/images/uploads/infrastructure-arch.png)

> ### Dataflow

Reviewer: ```suggestion
### [use case name] Dataflow
```

Reviewer: same

---

> ## [use case name] Architecture Diagram

Reviewer: ```suggestion
## Application Delivery and Operations Architecture Diagram
```

:::note
This flow assumes a service has been created for saving an HTML page containing a list of products. For further information, check the [Implementation](#implementation) section.
:::

![Orchestrator architecture diagram](/assets/docs/images/uploads/infrastructure-arch.png)

> ### [use case name] Dataflow

Reviewer: ```suggestion
### Application Delivery and Operations Dataflow
```
